### PR TITLE
feat(new-primary-school): School bus page

### PR DESCRIPTION
**Pull request by birkirkristmunds**
[TS-808](https://dit-iceland.atlassian.net/browse/TS-808)

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts [hunk 1]
```diff
@@ -616,7 +616,22 @@ export const NewPrimarySchoolForm: Form = buildForm({
...
           title:
             newPrimarySchoolMessages.differentNeeds.schoolBusSubSectionTitle,
-          children: [],
+          children: [
+            buildMultiField({
+              id: 'schoolBus',
+              title: newPrimarySchoolMessages.differentNeeds.schoolBusTitle,
+              children: [
+                buildAlertMessageField({
+                  id: 'schoolBusSubSection.info',
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:625]: Change id?
```suggestion
                  id: 'schoolBus.info',
```

**Review by veronikasif — APPROVED** [review]
Just one small comment, otherwise looks good! 💯

File: libs/application/templates/new-primary-school/src/lib/messages.ts [hunk 2]
```diff
@@ -461,6 +461,18 @@ export const newPrimarySchoolMessages: MessageDir = {
       defaultMessage: 'Skólaakstur',
       description: 'School bus',
     },
+    schoolBusTitle: {
+      id: 'dess.nps.application:different.needs.school.bus.title',
+      defaultMessage: 'Upplýsingar um skólaakstur',
+      description: 'Information about school bus',
+    },
+    schoolAlertMessage: {
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/lib/messages.ts:469]: Maybe change the name?
```suggestion
    schoolBusAlertMessage: {
```

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts (hunk 1 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:625]: Fixed

File: libs/application/templates/new-primary-school/src/lib/messages.ts (hunk 2 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/lib/messages.ts:469]: Fixed

**Pull request merged**